website/src/_includes/partials/footer.liquid

```diff
@@ -0,0 +1,11 @@
+<div class="footer-container mdl-grid">
+  <div class="mdl-cell mdl-cell--2-col mdl-cell--2-col-tablet mdl-cell--4-col-phone"><a href="https://pair.withgoogle.com/" target="_blank"><img src="{% root %}/assets/images/pair-logo.svg"/></a></div>
+  <div class="mdl-cell mdl-cell--2-col mdl-cell--2-col-tablet mdl-cell--4-col-phone"><a href="https://research.google/teams/brain/pair/" target="_blank">Google Research</a></div>
```

Comment:
We can link https://research.google/teams/language/ as well

Reply:
Removed the pair link and added language (the other pair link next to it is already a better link for PAIR)